nntrainer/tensor/blas_interface.cpp

```diff
@@ -400,12 +381,64 @@ void scopy_int8_to_float16(const unsigned int N, const uint8_t *X,
   copy_int8_to_fp16(N, X, incX, Y, incY);
 }
 
-void ewvm(const unsigned int N, const _FP16 *X, const _FP16 *Y, _FP16 *Z) {
-  ewvm_FP16(N, X, Y, Z);
+void ele_mul(const unsigned int N, const _FP16 *X, const _FP16 *Y, _FP16 *Z,
+             float alpha, float beta) {
+#if (defined USE__FP16 && USE_NEON)
+  nntrainer::neon::ele_mul(N, X, Y, Z, alpha, beta);
+#else
+  for (unsigned int i = 0; i < N; ++i) {
+    if (std::abs(beta) > __FLT_MIN__)
+      Z[i] = static_cast<_FP16>(alpha) * X[i] * Y[i] +
+             static_cast<_FP16>(beta) * Z[i];
+    else
+      Z[i] = static_cast<_FP16>(alpha) * X[i] * Y[i];
```

Comment:
any reason to differentiate beta == 0 case?
```suggestion
      Z[i] = static_cast<_FP16>(alpha) * X[i] * Y[i] + static_cast<_FP16>(beta) * Z[i];
```

Reply:
what if Z[i] != Z[i] ? (NaN in Z[i])

Reply:
regardless of beta, wouldn't it cause an error anyway?

Reply:
if beta is zero, Z = X * Y + beta * Z would be Z = X * Y.
if not, Z = X * Y + beta * Z.

Reply:
isn't NaN * 0 = NaN ?

Reply:
yeah, what I mean is having beta != 0 condition to avoid NaN or uninitialized error seems offbeat.

Reply:
maybe we could come up with a different way to handle these cases (e.g., check if tensor is initialized when using beta).

Reply:
Sounds more reasonable.
will try to fix it during resolving conflicts